libs/RegMgr/regmgr.cpp

```diff
@@ -1,4 +1,6 @@
-
+#ifdef _WIN32
```

Comment:
ifdef isn't needed here as linux only builds linregmgr.cpp

or could be moved to winregmgr.h under above winreg.h